noggin/defaults.py

```diff
@@ -1,5 +1,6 @@
 # This file contains the default configuration values
 import socket
+from flask_babel import _
```

Comment:
Black suggests adding a space before this line (you can check with `tox -e format`).